metricbeat/docs/running-on-kubernetes.asciidoc

```diff
@@ -200,41 +200,11 @@ Metrics should start flowing to Elasticsearch.
 
 
 [float]
-==== Deploying Metricbeat Daemonset with Leader Election enabled
-
-In some cases users may want to avoid deploying both a Deployment and a Daemonset
-to collect cluser-wide metrics and node-level metrics.
-For this case, we provide the option to deploy Metricbeat only as a Daemonset
-and leverage the leader election feature which allows to define configurations
-that are enabled only by the leader Pod. The leader Pod is automatically chosen
-between the Pods of the Daemonset.
-Here is an example of a configuration that enables leader election:
-[source,yaml]
------
-metricbeat.autodiscover:
-  providers:
-  - type: kubernetes
-    scope: cluster
-    node: ${NODE_NAME}
-    unique: true
-    identifier: leaderelectionmetricbeat
-    templates:
-      - config:
-          - module: kubernetes
-            hosts: ["kube-state-metrics:8080"]
-            period: 10s
-            add_metadata: true
-            metricsets:
-              - state_node
------
-Users can find more info about the `unique` and `identifier` options at <<configuration-autodiscover>>.
-
-Users can enable the respective parts the Daemonset ConfigMap and
-set the `replicas` of the Deployment to `0` in order to only deploy
-the Daemonset on the cluster with the leader election provider enabled
-in order to collect cluster-wide metrics:
-["source", "sh", subs="attributes"]
-------------------------------------------------
-curl -L -O https://raw.githubusercontent.com/elastic/beats/{branch}/deploy/kubernetes/metricbeat-kubernetes.yaml
-kubectl create -f metricbeat-kubernetes.yaml
-------------------------------------------------
+==== Deploying Metricbeat to collect cluster level metrics for big clusters
+
+In some cases, users might want to monitor really bog Kubernetes clusters. In such cases,
+the Pod that will be collecting cluster level metrics might face performance issues due to
+resources limitations. In this case users might consider avoid using leader election configuration
+for the Daemonset and instead deploy a singleton Metribceat instance using a
+https://github.com/elastic/beats/blob/7.9/deploy/kubernetes/metricbeat/metricbeat-deployment.yaml[Deployment] with the
+proper https://github.com/elastic/beats/blob/7.9/deploy/kubernetes/metricbeat/metricbeat-deployment-configmap.yaml[configuration].
```

Comment:
This is linking to older branches, we may easily forget to apply changes there that could be needed for future versions. Would it make sense to somehow copy the important parts of these manifests to some doc file? Or we just remove these links and leave to the user the definition of their configuration?

Reply:
🤔 My intention here is to point to the latest version that had been supporting the Deployment approach just for reference. I don't see any reason of updating this however do you see any case I might miss?

Reply:
If this is going to be the documentation for big clusters I think we should keep it updated. Imagine in 7.10/7.11, someone with a big cluster sees this section, and they use the 7.9 deployment linked here. They may end up using an incorrect version, and they may also be missing some configuration change that we haven't backported.

Reply:
👍  gotcha, thanks!